iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/InputFormatConfig.java

```diff
@@ -87,6 +87,8 @@ private InputFormatConfig() {
   public static final String CATALOG_DEFAULT_CONFIG_PREFIX = "iceberg.catalog-default.";
   public static final String QUERY_FILTERS = "iceberg.query.filters";
 
+  public static final String TABLE_PLAN_WORKER_POOL_SIZE = "iceberg.worker.num-threads";
```

Comment:
can we use iceberg WORKER_POOL config for that: 'iceberg-worker-pool'?

Reply:
If users don't set the `iceberg.worker.num-threads` in beeline console, we will use & follow the pool size which equals iceberg `WORKER_POOL`'s pool size, and `WORKER_POOL`'s pool size is also the default thread pool size in current iceberg design.

Reply:
can we ref the iceberg conf name like `SystemConfigs.WORKER_THREAD_POOL_SIZE.propertyKey()`

Reply:
Sure, we can. Fixed.

Reply:
👍